### ISSUE TITLE
Autocomplete list doesn't reman at proper position on the screen

### ISSUE THREAD

**Issue by mfendeksilverstripe** [silverstripe/silverstripe-framework#7923]
## Affected Version

4.0.x-dev

## Description

Components that use `autocomplete` functionality which comes from `JQuery UI` via `ui-autocomplete` class do not display properly when screen is scrolled. The problem is that the list of auto complete suggestions is displayed as an overlay with `position: absolute` property. This make the overlay stay at the same spot on the screen even if the actual auto-complete field as moved to a different position when page got scrolled.

![alt text](https://s17.postimg.org/ypw513epb/scroll_bug1.png "Logo Title Text 1")
![alt text](https://s17.postimg.org/osl481mj3/scroll_bug2.png "Logo Title Text 1")

## Steps to Reproduce

* use any auto-complete component for example `GridFieldAddExistingAutocompleter`
* input something in the auto-complete input field that triggers some values to be displayed as suggestions
* scroll the screen without losing focus of the auto-complete field

**Comment by chillu** [comment]
I'm marking this as `impact/low`, and we probably won't get around fixing this before moving the entire GridField to React, at which point it'll no longer be a jQuery autocomplete.

**Comment by mfendeksilverstripe** [comment]
@chillu I don't think this issue is related only to `GridField`. Another example would be the `AutocompleteField` which is a completely separate field and has the same issue.

https://github.com/tractorcow/silverstripe-autocomplete

My point is that any component that uses the JQuery UI auto-complete feature will have display issues as long as it is displayed in the content area of the CMS because of the way how the scroll works.

**Comment by frankmullenger** [comment]
@chillu is there a timeframe you are working to on this issue or the React GridField work?

**Comment by chillu** [comment]
@frankmullenger 3-6 months? There's a lot of moving parts in a React GridField. Is this regarded as a high impact bug by your client?

**Comment by frankmullenger** [comment]
@chillu not regarded as high impact, in our backlog for review along with some other small UX issues. 3-6 months seems like a reasonable timeframe from our perspective currently thanks.
